app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -46,11 +46,16 @@ def self.raw_connect(hostname, port, options)
   def self.openshift_connect(hostname, port, options)
     require 'openshift_client'
     api_endpoint = raw_api_endpoint(hostname, port)
-    osclient = OpenshiftClient::Client.new(api_endpoint, api_version)
-    # TODO: support real authentication using certificates
-    osclient.ssl_options(:verify_ssl => verify_ssl_mode)
-    osclient.bearer_token(options[:bearer]) if options[:bearer]
-    osclient
+
+    auth_options = {}
+    auth_options[:bearer_token] = options[:bearer] if options[:bearer]
```

Comment:
@simon3z any reason not to remove 2 lines above in favor of calling kubernetes_auth_options in ctor?
options[:username] && options[:password] will always be false.

This might be wrong if there is some reason why openshift client should NEVER use username an password as appose to kubeclient.